ports/mimxrt/boards/TEENSY40/TEENSY40_af.csv

```diff
@@ -0,0 +1,39 @@
+Pad,ALT0, ALT1, ALT2, ALT3, ALT4, ALT5, ALT6, ALT7, ALT8, ALT9,ADC,ACMP,Default
+GPIO_AD_B0_03,FLEXCAN2_RX,XBAR_INOUT17,LPUART6_RXD,USB_OTG1_OC,FLEXPWM1_PWM1_X,GPIO1_IO03,REF_CLK_24M,LPSPI3_PCS0,,,,ACMP4_IN4,ALT5
```

Comment:
can this board use one of the existing af files?  why does it need to be specific to the Teensy 4.0?

Reply:
Teensy uses different pins than the other boards and has more pins available to the user. So I think there needs to be a separate file.

Reply:
On stm32 there is one AF file per MCU and then different boards using the same MCU use the same AF file.  It doesn't matter if the board does not use all the pins, it just selects those pins from the AF file that it uses.

In other words: if board A and board B have the same i.MX MCU and expose the same pin (eg GPIO_AD_B0_03) does that pin have the exact same functions on both boards?  If so then they can share the same AF file.

Reply:
Mhmh yeah now I got your point. You are right. It makes sense to do it like that. I will fix that.

Reply:
Great!